### PR TITLE
[Docenhancement][Together mode][1827725]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Updated name change in teams together mode, toc, whats new, teams developer portal and teams apps in meetings.

### CODE REVIEW

File: msteams-platform/TOC.yml [hunk 1]
```diff
@@ -309,7 +309,7 @@
     href: apps-in-teams-meetings/create-apps-for-teams-meetings.md
   - name: Enable and configure apps for meetings
     href: apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md
-  - name: Together Mode in Teams
+  - name: Customize Together Mode Scenes in Teams
```
Comment (rarang) [inline msteams-platform/TOC.yml:312]: It is **CUSTOM** and NOT Customize

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 2]
```diff
@@ -20,13 +20,13 @@ The users can access apps during meetings using the tab gallery, for example:
 
 > [!VIDEO https://www.youtube-nocookie.com/embed/nKAy5rNDus4]
 
-This article provides an overview of meeting app extensibility, API references, enable and configure apps for meetings, and Together Mode in Teams.
+This article provides an overview of meeting app extensibility, API references, enable and configure apps for meetings, and customize Together Mode scenes in Teams.
```
Comment (rarang) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:23]: **custom** Together Mode scenes

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 3]
```diff
@@ -20,13 +20,13 @@ The users can access apps during meetings using the tab gallery, for example:
...
 
 You can enhance your meeting experience by using the meeting extensibility feature, which enables you to integrate your apps within meetings. It also includes different stages of a meeting lifecycle, where you can integrate tabs, bots, and messaging extensions. With meetings extensibility APIs, you can identify different participant roles and user types, get meeting events, generate in-meeting dialogs, and so on.
 
 To customize Teams with apps for meetings, you can enable your apps for Teams meetings by updating your app manifest and you can also configure your apps for meeting scenarios.
 
-The new Together Mode feature enables users to collaborate in a meeting with their team in one place without being separated by boxes.
+The new customize Together Mode scenes feature enables users to collaborate in a meeting with their team in one place without being separated by boxes.
```
Comment (rarang) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:29]: the word is **custom**

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 4]
```diff
@@ -35,7 +35,7 @@ The new Together Mode feature enables users to collaborate in a meeting with the
 * [Messaging extension](../messaging-extensions/what-are-messaging-extensions.md)
 * [Design your app](../apps-in-teams-meetings/design/designing-apps-in-meetings.md)
 * [Prerequisites and API references for apps in Teams meetings](create-apps-for-teams-meetings.md)
-* [Together Mode](~/apps-in-teams-meetings/teams-together-mode.md)
+* [Customize Together Mode scenes](~/apps-in-teams-meetings/teams-together-mode.md)
```
Comment (rarang) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:38]: **Custom**

**Review by rarang — DISMISSED** [review]

**Review by v-abirade — APPROVED** [review]
Approve

**Pull request merged**